assets/concepts/NutsOrganizationCredential.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "<<id>>@{1}",
+  "issuer": "<<issuer>>",
+  "type": "NutsOrganizationCredential@{2_1}",
```

Comment:
only fields are indexable? (so in combination with `<<....>>`), or did I miss something?

Reply:
yes, arrays are not (yet) supported. Unless the array contains a object ;)

Reply:
as defined in docs/pages/technology/vc-concepts.rst